### DISCUSSION TITLE
question: on network lan , if I just choose spool on job, will it go to the pool of printers?

### DISCUSSION THREAD

**Discussion by sidneyns** [smartin015/continuousprint#178] [General]
Hello.. 

I have 10 printers at home..8 in black and 2 in while filament. 

If I put on the lan Job an file... can I just specify black and it will queue on all black printers?  When the next printer comes available it gets from the queue and starts to print?  

This way is great to optimize the printers.

**Comment by simoncrabbuk** [comment]
Would naming the LAN queues on the different machines work?  BLACKLAN WHITELAN. Give it a try, I'm hoping it works like that.

**Comment by smartin015** [comment]
Hey @sidneyns, @simoncrabbuk!

I haven't tested it myself, but [material selection](https://smartin015.github.io/continuousprint/material-selection/) will likely work on LAN queues the same way it works with local queues - currently, that behavior is to go sequentially down the queue and print each job, stopping for a filament change if there's a color that isn't loaded onto the printer. I anticipated LAN printers needing different [strategies](https://smartin015.github.io/continuousprint/lan-queues/#queue-strategies) for handling queue jobs, but haven't yet expanded to anything else yet. 

@simoncrabbuk's solution of multiple LAN queues should work in the meantime - with your black filament printers joined to `BLACKLAN` and your white filament printers to `WHITELAN`, they will operate separately on the two queues. Don't have both joined to both queues though - otherwise, they'll pick from either. 

I'm sure there are other fun/interesting intelligent things that could be done with queue ordering. Given the interest, I opened #179 to collect ideas for future implementation :)
